src/main/scala/cognite/spark/v1/Constants.scala

```diff
@@ -5,6 +5,7 @@ import BuildInfo.BuildInfo
 import scala.concurrent.duration._
 
 object Constants {
+  val CreateDataPointsLimit = 10000
```

Comment:
Why 10,000 when the limit is higher?

https://docs.cognite.com/api/v1/#operation/postMultiTimeSeriesDatapoints
>  The limit per request is 100000 datapoints.

Reply:
Ohh, good catch! I thought it was 10k from this part
>Array of DatapointsWithInternalId (object) or DatapointsWithExternalId (object) (DatapointsPostDatapoint) [ 1 .. *10000* ] items
 